src/main/java/Managers/Bao.java

```diff
@@ -0,0 +1,209 @@
+package Managers;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+import Components.Task;
+
+import Exceptions.BadDateTimeFormatException;
+import Exceptions.BadIndexException;
+import Exceptions.BaoUnknownCommand;
+import Exceptions.MaxTaskException;
+import Exceptions.NoDateTimeException;
+import Exceptions.NoTaskDescriptionException;
+
+import static Constants.BaoConstants.LOGO;
+import static Constants.BaoConstants.LINE_BREAK;
+
+public class Bao {
+    private static Scanner in = new Scanner(System.in);
```

Comment:
Maybe you should consider using more descriptive names like "inScanner" for this object?